docs/Errors.md

```diff
@@ -173,3 +173,33 @@ You cannot use `send` inside the `onError` hook.
 #### FST_ERR_SEND_UNDEFINED_ERR
 
 Undefined error has occurred.
+
+<a name="FST_ERR_PLUGIN_NOT_VALID"></a>
+#### FST_ERR_PLUGIN_NOT_VALID
+
+Plugin must be a function or a promise.
+
+<a name="FST_ERR_PLUGIN_TIMEOUT"></a>
+#### FST_ERR_PLUGIN_TIMEOUT
+
+Plugin did not start in time.
```

Comment:
Could we share the timeout default here?

Reply:
Done in d315fed